code/style.css

```diff
@@ -1,4 +1,408 @@
+  font-weight: 600;
+  padding: 20px;
+}
+
+.left-links:hover, .right-links:hover {
+  opacity: 0.6;
+}
+
+.logo {
+  width: 150px;
+  height: 150px;
+  padding: 10px;
+}
+
+
+/* ------BIG NEWS SECTION STYLING ----- */
+
+.big-news {
+  display: flex;
+  flex-direction: column;
+  margin: 30px;
+  padding: 20px;
+  background-color: #fff;
+}
+
+.big-news-text {
+  font-weight: lighter;
+  font-size: 16px;
+}
+
+hr {
+  height: 5px;
+  border: none;
+  background-color: rgb(59, 55, 55);
+  margin: 30px;
+}
+
+.big-news a {
+  text-decoration: none;
+  color: grey;
+}
+
+.big-news-image {
+  width: 100%;
+}
+
+.big-news:hover {
+  -webkit-transform: scale(1.005);
+  -moz-transform: scale(1.005);
+  transform: scale(1.005);
+  -webkit-box-shadow: 0 10px 20px rgba(0, 0, 0, .7);
+  -moz-box-shadow: 0 10px 20px rgba(0, 0, 0, .7);
+  box-shadow: 0 10px 20px rgba(0, 0, 0, .7);
+}
+
+.top-news-text {
+  color: #bababa;
+  font-size: 2rem;
+  box-sizing: border-box;
+  text-decoration: none;
+  margin: 50px;
+}
+
+.top-projects:hover {
+  color: black;
+}
+
+/* ------OTHER NEWS SECTION STYLING ----- */
+
+h4 {
+  position: absolute;
+  font-style: italic;
+  text-align: center;
+  background-color: #C9D6DF90;
+  color: black;
+  padding: 10px;
+  box-shadow: 8px 8px yellow;
+  width: 100px;
+  font-size: 20px;
+  margin: 0;
+}
+
+.news-cards {
+  /* width: 100%;  If I put a margin to news-card the page does not fit in the browser*/
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  text-align: justify;
+  margin: 30px;
+}
+
+.card {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  margin: 20px;
+  padding: 15px;
+  border: 1px solid #F0F5F9;
+}
+
+.card a {
+  text-decoration: none;
+  color: grey;
+}
+
+.news-image {
+  width: 100%;
+  display: flex;
+  align-self: center;
+}
+
+.card:hover {
+  -webkit-transform: scale(1.005);
+  -moz-transform: scale(1.005);
+  transform: scale(1.05);
+  -webkit-box-shadow: 0 10px 20px rgba(0, 0, 0, .7);
+  -moz-box-shadow: 0 10px 20px rgba(0, 0, 0, .7);
+  box-shadow: 0 10px 20px rgba(0, 0, 0, .7);
+}
+
+
+/* ------CTA SECTION STYLING ----- */
+
+.cta {
+  background-color: #fff700;
+  padding: 3% 5%;
+  text-align: center;
+}
+
+.cta-text {
+  font-family: 'Bebas Neue', cursive;
+  font-size: 1.5rem;
+  line-height: 1.5;
+}
+
+/* ------subscribtion button & animation ----- */
+
+button, button::after {
+  width: 200px;
+  height: 66px;
+  font-size: 24px;
+  font-family: 'Bebas Neue', cursive;
+  background: linear-gradient(45deg, transparent 5%, black 5%);
+  border: 0;
   color: #fff;
+  letter-spacing: 3px;
+  line-height: 66px;
+  box-shadow: 6px 0px 0px #00E6F6;
+  outline: transparent;
+  position: relative;
+}
+
+button::after {
+  --slice-0: inset(50% 50% 50% 50%);
+  --slice-1: inset(80% -6px 0 0);
+  --slice-2: inset(50% -6px 30% 0);
+  --slice-3: inset(10% -6px 85% 0);
+  --slice-4: inset(40% -6px 43% 0);
+  --slice-5: inset(80% -6px 5% 0);
```

Comment:
What do these do?

Reply:
It's about the animation for the subscribe button. I don't know the exact explanation but it somehow slices the button.